### PR TITLE
Send X-Request-ID with every request ...

### PR DESCRIPTION
**Pull request by DeepDiver1975**
refs #140 

blocked by https://github.com/owncloud/core/pull/33909

### CODE REVIEW

**Comment by PVince81** [comment]
how to fix CI ? you'd need the docker in Travis to point at stable10

**Comment by DeepDiver1975** [comment]
No idea how to handle this. We should only send the Header with 10.1 and higher

**Comment by PVince81** [comment]
You can detect version with capabilities I believe

**Comment by DeepDiver1975** [comment]
So the capabilities requests never sends the header then .... Sounds reasonable

**Comment by DeepDiver1975** [comment]
@PVince81 mind reviewing this one again? THX

File: src/helperFunctions.js [hunk 1]
```diff
@@ -140,6 +142,25 @@ helpers.prototype._updateCurrentUser = function () {
   })
 }
 
+helpers.prototype.buildHeaders = function () {
+  var headers = {
+    authorization: this._authHeader,
+    'OCS-APIREQUEST': true
+  }
+  if (this.atLeastVersion('10.1.0')) {
```
Comment (PVince81) [inline src/helperFunctions.js]: hmm ok so no capability then ?

**Pull request merged**